<ui version="4.0" >
 <class>ImportTrainingTextLocalPage</class>
 <widget class="QWidget" name="ImportTrainingTextLocalPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="lbDesc" >
     <property name="text" >
      <string>Hier können Sie Textdateien importieren. Sollte ihr Text in
einem anderen Format (z.B.: einem Word Dokument)
vorliegen, kännen Sie ihn in der spezifischen Applikation
öffnen, als Text speichern und diese Datei hier angeben.

Bitte wählen Sie nun die Datei aus die Sie gerne
importieren möchten:</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Name des Textes:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="leName" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Datei:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLineEdit" name="lePath" />
       </item>
       <item>
        <widget class="QToolButton" name="tbSelectPath" >
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../action_icons.qrc" >:/images/icons/document-open.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../action_icons.qrc" />
 </resources>
 <connections/>
</ui>
